src/widgets/DeviceActionButton/DeviceActionButton.jsx

```diff
@@ -50,15 +51,19 @@ const DeviceActionButton = ({
         icon: '',
     };
 
+    const openDoor = debounce(execute, 2000);
```

Comment:
Please keep the timeout at 3000, since the door mechanisms are open for at least 3 seconds and there is no reason to request opening again. Also this is a generic function that is used for both doors and lights (and other things in the future) so please either use a more generic name like executeDelayed or limit debouncing to just the door device type.